content/en/dora_metrics/data_collected/_index.md

```diff
@@ -63,7 +63,14 @@ Datadog breaks down change lead time into the following metrics, which represent
 
 These metrics are only computed when the source of the repository metadata is GitHub, and there must be a pull request (PR) associated with a commit, if any. A commit is associated with a PR if the commit is first introduced to the target branch when merging that PR. If a commit does not have an associated PR, only `dora.time_to_deploy` and `dora.deploy_time` metrics are available.
 
-**Note:** These metrics are emitted for every commit and not per deployment.
+**Notes:**
+
+- These metrics are emitted for every commit and not per deployment.
+- Commits without meaningful changes (for example, merge commits) are ignored.
+- For rebased feature branches, *change lead time* calculations consider the new commits created during the rebase, not the original commits.
```

Comment:
```suggestion
- For rebased branches, *change lead time* calculations consider the new commits created during the rebase, not the original commits.
```
It would also be true if you do a rebase on the main/deployment branches (nobody would do this, but still)